### PR TITLE
[ConvectionDiffusionApplication] Solving tests issues

### PR DESCRIPTION
**Pull request by loumalouomega**
Updating processes, correcting solver

Additionally clean up of the check tetrahedral mesh

### CODE REVIEW

**Comment by jcotela** [comment]
:+1: for the changes in tetrahedral_mesh_orientation_check, but I'll wait for someone to weigh in the other parts of the code before approving.

File: applications/convection_diffusion_application/python_scripts/convection_diffusion_base_solver.py [hunk 1]
```diff
@@ -164,11 +162,7 @@ def __init__(self, model, custom_settings):
 
         self.print_on_rank_zero("::[ConvectionDiffusionBaseSolver]:: ", "Construction finished")
 
-    def AddVariables(self, target_model_part=None):
```
Comment (rubenzorrilla) [inline applications/convection_diffusion_application/python_scripts/convection_diffusion_base_solver.py]: If I'm not wrong, this was required for the CHT solver. Are you sure of this change?

File: applications/convection_diffusion_application/python_scripts/convection_diffusion_base_solver.py [hunk 2]
```diff
@@ -164,11 +162,7 @@ def __init__(self, model, custom_settings):
...
-
-        if target_model_part == None:
```
Comment (rubenzorrilla) [inline applications/convection_diffusion_application/python_scripts/convection_diffusion_base_solver.py]: If I'm not wrong, this was required for the CHT solver. Are you sure of this change?

**Review by rubenzorrilla — CHANGES_REQUESTED** [review]

**Review by rubenzorrilla — APPROVED** [review]
Thanks @loumalouomega

**Pull request merged**